io.welle.welle-gui.yml

```diff
@@ -0,0 +1,103 @@
+app-id: io.welle.welle-gui
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: welle-io
+finish-args:
+  - '--share=network'
+  - '--share=ipc'
+  - '--filesystem=xdg-documents'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  - '--device=dri'
+  - '--device=all'
+
+modules:
+  - name: fftw
+    buildsystem: autotools
+    config-opts:
+      - '--enable-shared'
+      - '--disable-static'
+      - '--enable-threads'
+      - '--enable-float'
+    sources:
+      - type: archive
+        url: 'http://fftw.org/fftw-3.3.10.tar.gz'
+        md5: 8ccbf6a5ea78a16dbc3e1306e234cc5c
+
+  - name: libfaad
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: 'https://salsa.debian.org/multimedia-team/faad2.git'
+        tag: debian/2.11.2-1
+
+  - name: libusb
+    buildsystem: autotools
+    config-opts:
+      - '--disable-udev'
+    sources:
+      - type: git
+        url: 'https://github.com/libusb/libusb.git'
+        tag: v1.0.27
+
+  - name: librtlsdr
+    buildsystem: cmake
+    config-opts:
+      - '-Wno-dev'
+      - '-DDETACH_KERNEL_DRIVER=ON'
+    sources:
+      - type: git
+        url: 'https://gitea.osmocom.org/sdr/rtl-sdr.git'
+        tag: v2.0.2
+
+  - name: libairspy
+    buildsystem: cmake
+    config-opts:
+      - '-Wno-dev'
+    sources:
+      - type: git
+        url: 'https://github.com/airspy/airspyone_host.git'
+        tag: v1.0.10
+
+  - name: SoapySDR
+    buildsystem: cmake
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-Wno-dev'
+    sources:
+      - type: git
+        url: 'https://github.com/pothosware/SoapySDR.git'
+        tag: soapy-sdr-0.8.1
+
+  - name: welle.io
+    buildsystem: qmake
+    build-commands:
```

Comment:
build-commands is only for when `buildsystem` is `simple`

Reply:
Understood. But I need this, currently (see https://github.com/flathub/flathub/pull/6274#discussion_r1991000223).

Reply:
it won't work together with buildsystem: qmake, if you need it drop it and run qmake manually with the proper arguments.

Reply:
a quick search https://forum.qt.io/topic/8774/define-the-installation-prefix-using-qmake

Reply:
Ok, I will try to use the PREFIX.

Reply:
Fixed in https://github.com/AlbrechtL/flathub/commit/fb66dbf0986effa9ed6f7e80d1cd6bede07ab510